build.rs

```diff
@@ -64,7 +66,7 @@ const RING_SRCS: &[(&[&str], &str)] = &[
     (&[X86_64], "crypto/fipsmodule/ec/asm/p256-x86_64-asm.pl"),
     (&[X86_64], "crypto/fipsmodule/modes/asm/aesni-gcm-x86_64.pl"),
     (&[X86_64], "crypto/fipsmodule/modes/asm/ghash-x86_64.pl"),
-    (&[X86_64], "crypto/poly1305/poly1305_vec.c"),
```

Comment:
All this code is guarded by `#if defined(BORINGSSL_HAS_UINT128) && defined(OPENSSL_X86_64)
` so adding it for WebAssembly targets doesn't make sense.